src/locales/de/dialogue.ts

```diff
@@ -482,7 +482,7 @@ export const PGMdialogue: DialogueTranslationEntries = {
       3: "Was machst du hier? Bist du uns gefolgt?"
     },
     "victory": {
-      1: "Na gut, bis der Boss Zeit für dich hat, werde ich dein Gegner sein!",
+      1: "Muwuhahaha! That battle was fun even though I lost!",
```

Comment:
```suggestion
      1: "Bwaarharharharhar!! Ich hab zwar verloren, aber das hat Spaß gemacht!",
```
https://abcboy101.github.io/poke-corpus/#id=oras.common.23.2332